modin/core/execution/ray/common/utils.py

```diff
@@ -147,6 +149,14 @@ def initialize_ray(
     else:
         NPartitions._put(num_cpus)
 
+    # TODO(https://github.com/ray-project/ray/issues/28216): remove this
+    # workaround once Ray gives a better way to suppress task errors.
+    # Ideally we would not set global environment variables.
+    # If user has explicitly set _RAY_IGNORE_UNHANDLED_ERRORS_VAR, don't
+    # don't override its value.
+    if _RAY_IGNORE_UNHANDLED_ERRORS_VAR not in os.environ:
+        os.environ[_RAY_IGNORE_UNHANDLED_ERRORS_VAR] = "1"
```

Comment:
LGTM If ray checks this value not only at startup.

Reply:
Ray does seem to do that. This PR fixes #5016 even with the latest code.